src/styles/parts/appName.sass

```diff
@@ -0,0 +1,3 @@
+.appName
+	padding-left: 0.3em
+	border-left: 0.3em solid $c-main
```

Comment:
To, že jsi zahodil manifest, mě dost mrzí. :( Co já vím, tak přítomnost manifestu například zvyšuje hodnocení v Googlu. Čím více ten manifest integrujeme do zbytku examplu, tím je jistější, že bude v každém projektu z něj vycházejícím dostatečně udržovaný.